Set the terminal window's "class".  This is a workaround that allows JWM to
properly  match the terminal window.  JWM doesn't seem to re-evaluate groups
when window's title is updated.
--- a/unix/window.c	2022-11-07 22:43:12.703088650 -0500
+++ b/unix/window.c	2022-11-07 23:00:05.808991945 -0500
@@ -5328,7 +5328,11 @@
     inst->window = make_gtk_toplevel_window(inst);
     gtk_widget_set_name(GTK_WIDGET(inst->window), "top-level");
     {
+#if 1 // This is a workaround for JWM not properly matching the window.
+        const char *winclass = "PuTTY Terminal";
+#else
         const char *winclass = conf_get_str(inst->conf, CONF_winclass);
+#endif
         if (*winclass) {
 #if GTK_CHECK_VERSION(3,22,0)
 #ifndef NOT_X_WINDOWS
